stylesheets/scss/layouts/_two-columns.scss

```diff
@@ -9,6 +9,7 @@
   .left,
```

Comment:
When I scroll down the map goes over the navbar, what should I do to change this?

Reply:
Please let me know once you've implemented the changes so I can check what's the issue. I won't be able to check it before tomorrow, though.